clients/feeder/testdata/sepolia-integration/transaction/0x1111.json

```diff
@@ -1,7 +1,7 @@
 {
     "revert_error": "some error",
-    "execution_status": "REJECTED",
-    "finality_status": "RECEIVED",
+    "finality_status": "ACCEPTED_ON_L2",
+    "execution_status": "REVERTED",
```

Comment:
Mempool is not available yet, so technically Rejected is not possible. We should instead test the Reverted case for now.

Reply:
I am a bit out of context here. Could you provide more information so I know what you mean?

Reply:
After I double checked again, Rejected is still available, so I revert this change and add a new test for reverted case.